specification/nhs-website-content-api.yaml

```diff
@@ -52,16 +59,41 @@ info:
     All pages under these urls also have endpoints
 
     ## Network access
-    No special network access is required for this API.
+    No special network access is required for this API. It is available over the internet.
 
     ## Security and authorisation
-    You will need to provide a subscription key in the header or as a query parameter in your requests.
+    This API is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
-    Your subscription key can be found by logging into the API portal (provide link)
+    The end user will need to provide a subscription key in the header or as a query parameter in requests.
```

Comment:
In the new API Platform 'a subscription key' are referred to as 'an API key'.
Maybe we should update this to match.